subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1667,6 +1683,15 @@ static int init_reset(void)
 	mem_init(conn_pool, sizeof(struct ll_conn),
 		 sizeof(conn_pool) / sizeof(struct ll_conn), &conn_free);
 
+	/* Invalidate connection handles, refer to ll_connected_get() */
+	for (uint16_t handle = 0U; handle < CONFIG_BT_MAX_CONN; handle++) {
+		struct ll_conn *conn;
+
+		/* handle in valid range, conn will be non-NULL */
+		conn = ll_conn_get(handle);
+		conn->lll.handle = LLL_HANDLE_INVALID;
```

Comment:
The changes in this PR implies that `ll_conn_get` can now return NULL, which is missing a check here before accessing the pointer on the next line